packages/notifi-dapp-example/src/app/notifi/dashboard/page.tsx

```diff
@@ -41,17 +44,48 @@ export default function NotifiDashboard() {
 
   return (
     <div className="min-h-screen flex items-start flex-row">
+      {/* desktop sidebar */}
       <DashboardSideBar
         account={account}
         cardView={cardView}
         setCardView={setCardView}
+        className="md:block hidden"
```

Comment:
Do we have to pass the class name here? or we can just put it in component level.